utils/test/jest.config.js

```diff
@@ -26,7 +26,7 @@ module.exports = {
   testEnvironment: 'jest-environment-jsdom',
   setupFilesAfterEnv: ['<rootDir>/utils/test/jest.setup.js'],
   modulePathIgnorePatterns: ['./node_modules'],
-  transformIgnorePatterns: ['\\/node_modules\\/(?!@zendeskgarden|@babel)'],
+  transformIgnorePatterns: ['/node_modules/(?!(@zendeskgarden|@babel|react-merge-refs))'],
   transform: {
     '^.+\\.(t|j)sx?$': [
```

Comment:
```suggestion
    '^.+\\.(t|j|mj)sx?$': [
```
Including `mj` into the capture group will match `mjs` files.

Reply:
Unfortunately that didn't seem to do the trick. :(

Reply:
I also tried: `'^.+\\.(t|j|mj)s?(x)?$'` to capture non-`sx` files (e.g., `mjs`) but no bueno.